src/Features/CSharp/Portable/CodeRefactorings/EnableNullable/EnableNullableCodeRefactoringProvider.FixAllProvider.cs

```diff
@@ -36,7 +36,8 @@ public override IEnumerable<FixAllScope> GetSupportedFixAllScopes()
                 Debug.Assert(fixAllContext.Scope == FixAllScope.Solution);
                 return Task.FromResult<CodeAction?>(new FixAllCodeAction(EnableNullableReferenceTypesInSolutionAsync));
 
-                async Task<Solution> EnableNullableReferenceTypesInSolutionAsync(CodeActionPurpose purpose, CancellationToken cancellationToken)
+                async Task<Solution> EnableNullableReferenceTypesInSolutionAsync(
+                    CodeActionPurpose purpose, IProgress<CodeAnalysisProgress> progress, CancellationToken cancellationToken)
```

Comment:
a good location where we could actually update the progress as we're fixing the entire solution.